.github/workflows/book.yml

```diff
@@ -48,9 +49,9 @@ jobs:
       - uses: aws-actions/configure-aws-credentials@v4.0.2
         if: github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-1
+          role-to-assume: arn:aws:iam::024603541914:role/GitHubOIDCRole
+          role-session-name: S2nQuicGHABookSession
```

Comment:
is there a benefit to having a different `role-session-name` for every workflow, or could they all be the same? And does the first letter need to be capitalized like that?

Reply:
They can all be the same, but the session-name could be used in a future policy change to restrict an action from assuming a role (e.g. we want to separate s3 uploads from ECR uploads). I'm trying to leave the door open for more restrictions, but I'm open to simplifying it for v1.

Reply:
Ok, pushed a simplified naming scheme.